consensus/polybft/consensus_runtime.go

```diff
@@ -108,16 +108,25 @@ type consensusRuntime struct {
 	// checkpointManager represents abstraction for checkpoint submission
 	checkpointManager *checkpointManager
 
+	// proposerCalculator is the object which calculates new proposer
+	proposerCalculator ProposerCalculator
```

Comment:
```suggestion
	proposerCalculator *ProposerCalculator
```

Reply:
This is fixed as ProposerCalculator interface was removed: https://github.com/0xPolygon/polygon-edge/pull/1018/commits/48f6b5d8b23ffd0eb1cfc40fe6aa9ccfd9188845